src/routes/MetaDetails/styles.less

```diff
@@ -26,7 +27,11 @@
         z-index: 0;
         display: flex;
         flex-direction: row;
-        justify-content: space-between;
+
+        .vertical-nav-bar {
+            flex: none;
+            width: var(--vertical-nav-bar-size);
```

Comment:
this width is not needed anymore.
can you put the `--vertical-nav-bar-size` value here?